cmd/broker/main.go

```diff
@@ -51,6 +51,14 @@ func run() error {
 	return runWithContext(ctx)
 }
 
+const(
```

Comment:
If we are creating these constants can we ensure they are used where needed elsewhere in the code?